src/lib/components/Pagination/index.tsx

```diff
@@ -1,7 +1,9 @@
 import classNames from 'classnames';
 import type { ComponentProps, FC, PropsWithChildren } from 'react';
 import { HiChevronLeft, HiChevronRight } from 'react-icons/hi';
+import { excludeClassName } from '../../helpers/exclude';
 import range from '../../helpers/range';
+import { useTheme } from '../Flowbite/ThemeContext';
 
 export type PaginationProps = PropsWithChildren<Pagination>;
 interface Pagination extends ComponentProps<'nav'> {
```

Comment:
I keep forgetting to do this, we should Omit className so users will get a TS error for trying to add one